txpipe/selector.py

```diff
@@ -485,13 +538,19 @@ def select(self, data, bin_index):
         s2n_cut = self.config['s2n_cut']
         T_cut = self.config['T_cut']
         verbose = self.config['verbose']
-
-        s2n = data['mcal_s2n']
-        T = data['mcal_T']
+        
+        if self.config['shear_type']=='metacal':
```

Comment:
To avoid spiralling complexity in the downstream stages I think we need to rename the metacal columns after loading them, to remove the mcal prefix and remove lots of the `if type === metacal` things.

I know that's a pain - sorry.

Reply:
I tried to take out any of the if type == metacal pertaining to the column names themselves, and left the column names without the mcal prefix - is there a section in the inputs that needs to be added to rename the columns?